src/ducks/categories/helpers.js

```diff
@@ -51,6 +51,10 @@ export const getCategoryId = transaction => {
     return transaction.cozyCategoryId
   }
 
+  if (!transaction.localCategoryId && !transaction.cozyCategoryId) {
+    return null
+  }
+
   return transaction.automaticCategoryId
```

Comment:
Maybe we can add a comment, "returned only if the automatic categorization by Cozy is not deemed not usable as per GLOBAL_MODEL_USAGE_THRESOLD" ?